oak-store-document/src/main/java/org/apache/jackrabbit/oak/plugins/document/VersionGarbageCollector.java

```diff
@@ -1613,8 +1636,8 @@ private int removeUnusedPropertyEntries(NodeDocument doc,
                 // could be due to node not existing or current value being in a split
                 // doc - while the former is unexpected, the latter might happen.
                 // in both cases let's skip this property
-                if (log.isDebugEnabled()) {
-                    log.debug("removeUnusedPropertyEntries : no visible revision for property {} in doc {}",
+                if (DETAILED_GC_LOG.isDebugEnabled()) {
+                    DETAILED_GC_LOG.debug("removeUnusedPropertyEntries : no visible revision for property {} in doc {}",
```

Comment:
I guess we should decide how we treat property names - but this one is similar to the others and I think it should be log.trace